### DISCUSSION TITLE
Is it possible to opt out of dot notation or escape dot for field names?

### DISCUSSION THREAD

**Discussion by chrisirhc** [TanStack/form#532] [Q&A]
I'm considering migrating a project this library but there are field names containing dots `x.y` but these aren't to indicate nested fields.
Is there a way to opt-out of the path behavior? I do see the value in this feature but could there be a capability to escape the field names perhaps (e.g. `x\.y\.z`)?

I'm happy to contribute this feature but want to understand if any interest.

**Comment by crutchcorn** [comment]
Closing in favor of #639
